x-pack/plugins/cloud/server/routes/elasticsearch_routes.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { Logger } from '@kbn/logging';
+import { ElasticsearchConfigType } from '../../common/types';
+import { ELASTICSEARCH_CONFIG_ROUTE } from '../../common/constants';
+
+export function defineRoutes({
+  elasticsearchUrl,
+  logger,
+  router,
+}: {
+  elasticsearchUrl?: string;
+  logger: Logger;
+  router: IRouter;
+}) {
+  router.get(
+    {
+      path: ELASTICSEARCH_CONFIG_ROUTE,
+      validate: {},
+    },
```

Comment:
Should this route have `access: 'internal',` ?

```suggestion
    {
      path: ELASTICSEARCH_CONFIG_ROUTE,
      validate: {},
      options: {
        access: 'internal',
      },
    },
```